graspologic/cluster/autogmm.py

```diff
@@ -687,8 +720,18 @@ def _process_paramgrid(paramgrid):
             ag_params = {key: params[key] for key in ag_keys}
             if ag_params not in ag_params_processed:
                 ag_params_processed.append(ag_params)
+            if (
+                gm_params not in gm_params_processed
+                and ag_params["affinity"] == "none"
+                and run_multiple_init
+            ):
+                for _ in range(n_init):
+                    gm_params_processed.append(gm_params.copy())
+                    gm_params_processed[-1].update({"n_init": 1})
 
             paramgrid_processed.append([ag_params, gm_params])
+    for pa in gm_params_processed:
```

Comment:
Maybe you can simplify and merge the this for loop and the one on line 728 and then you can even remove the variable `gm_params_processed`

Reply:
Thank you for the feedback! I should have updated the code by moving this loop before the call of `process_paramgrid`.

Reply:
Actually, I was saying that in this [commit](https://github.com/microsoft/graspologic/pull/662/commits/498b2edcff0e7e9a0b5a2e25e0af977735e039a0) maybe it's simpler to drop the for loop 
`for pa in gm_params_processed` (on line733)
so that after the if condition 
`if ag_params["affinity"] == "none" and run_multiple_init` (on line 723), 
you could do those things at once: 1) make `n_init` copies of `gm_params`; 2) add `{"n_init": 1}` to each of them; 3) add `{"affinity": "none"}` to each of them

Reply:
@tliu68 I could do this! the only reason why I had the `for pa in gm_params_processed` in the end was because that way the new rows would be appended at the end of the dataframe rather than in between. I am fine changing it back to this implementation and doing it this other way, just let me know what you would prefer. Ideally, would like to get this merged by May 6 if possible.

Reply:
I see, okay, if you think that would make it easier to interpret/ check the results, then I'm cool with it :)